modules/apps/powwow/powwow-portlet/docroot/meetings/js/recurrence.js

```diff
@@ -410,6 +413,48 @@
 							startDateDatePicker.after('selectionChange', A.bind(instance._onStartDateDatePickerChange, instance));
 						},
 
+						_bindIntervalSelection: function() {
+							var instance = this;
+
+							var frequencyValue = instance._getFrequency();
+
+							var interValContainer = instance.get('intervalSelect');
+							interValContainer.all('option').remove();
+
+							var intervalCount = 0;
+
+							if (frequencyValue == FREQUENCY_DAILY) {
+								intervalCount = 90;
```

Comment:
i just found that currently the Powwow meeting support maximum 30 for daily interval. i think we can keep this value of 30. i'll also need to update the requirements in tickets for this.